web/templates/popovers/stream_info_popover.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="popover-menu stream-info-popover" data-simplebar data-simplebar-tab-index="-1">
+    <ul role="menu" class="popover-menu-list">
+        <li role="none" class="link-item popover-menu-list-item">
```

Comment:
We use `.link-item` class for the popover menu options. We should use `.text-item` here.

Reply:
Corrected it in the last commit, thanks!